doc/metadata_signing.md

```diff
@@ -0,0 +1,43 @@
+#Proxy Node Metadata Signing
+
+ [Proxy Node metadata](https://proxy-node.eidas.test.london.verify.govsvc.uk/ServiceMetadata) is generated by the [Verify Metadata Controller](https://github.com/alphagov/verify-metadata-controller), from a kubeyaml declaration of
+`kind: Metadata`.
+
+The metadata is signed with a CloudHSM key and its accompanying metadata signing certificate is chained to the following roots of trust in prodution or test:
+
+````
+Production
+├── verify-root-ca-a1
+│   └── proxy-node-metadata-ca-a1
+│       └── verify-eidas-proxy-node-prod-metadata-signing-cert-a1
+Test
+└── verify-root-ca-test-a1
+    └── proxy-node-metadata-ca-test-a1
+        ├── verify-eidas-proxy-node-build-metadata-signing-cert-a1
+        └── verify-eidas-proxy-node-integration-metadata-signing-cert-a1
+````
+
+The nodes in this chain relate to [Kubernetes secrets](https://kubernetes.io/docs/concepts/configuration/secret/) in the GSP Verify cluster. The pem-encoded certificate can be found on the `cert` key for each secret.
+
+Each certificate is declared with kubeyaml of
+ `kind: CertificateRequest`. A certificate can define its parent by secret name.
+ 
+ The root CAs are defined in the `verify-metadata-controller` repository.
+ 
+ ## CloudHSM Keys for certificates
+ Each certificate is created with a CloudHSM keypair with labels:
+ ````
+${namespace}-${secret}
+ ${namespace}-${secret}:public
+````
+for the private and public key respectively. These CloudHSM keys cannot be deleted.
+
+## Deleting and generating certificates
+Certificates can be deleted by deleting the Kubernetes secret that holds them.
+
+If the kubeyaml of `kind: CertificateRequest` is retained in the cluster namespace with the same name, the certificate and secret will be regenerated using the existing CloudHSM keypair.
+
+The `verify-metadata-controller` pod `v1-vmc-0` can be cycled for this regeneration.
```

Comment:
Is it always going to be `v1-vmc-0` (pod 0 specifically)?

Reply:
It's always been that name since it existed so seems safe to refer to it by name. I can make it more general by referring to the vmc pod though.